oddbird/oddleventy
# [winging-it] Add Winging it Index

![](https://source.unsplash.com/featured/?cute,animal&afdafs)

## Description
Add an Archive page and single view show winging it shows

## Steps to test/reproduce
- View archive (should be at /wingingit as we advertised)
- View single show
- View the /tags archive, and see the Winging It featured tag

https://deploy-preview-378--oddleventy.netlify.app/wingingit
https://deploy-preview-378--oddleventy.netlify.app/tags
https://deploy-preview-378--oddleventy.netlify.app/2023/07/09/winging-it-01/

## Show me

Winging It List Page:

![landing](https://github.com/oddbird/oddleventy/assets/1581694/9ebe2bc5-e390-4fa4-8405-67916e12993e)


Winging It Episode Single:

![single](https://github.com/oddbird/oddleventy/assets/1581694/9db9a3d3-0096-4d79-bfde-535123f18465)

Special Tag Section: 

![hover-over-highlighted-tag](https://github.com/oddbird/oddleventy/assets/1581694/a910b66c-a99a-4b24-92bb-740c91aea8e7)


Shows up in Posts List: 

![Post-list](https://github.com/oddbird/oddleventy/assets/1581694/c8c6c13a-b39c-4f38-ad55-9bf4f76b708b)
